src/app/docs/metric.js

```diff
@@ -23,5 +23,27 @@ angular
         $scope.parents = dag_utils.getParents(project, metric);
         $scope.parentsLength = $scope.parents.length;
 
+        $scope.versions = {
+            'Definition': codeService.generateMetricSQL($scope.metric)
+        }
+
+        let metric_type;
+        if ($scope.metric.type == 'expression') {
+            metric_type = 'Expression metric';
+        } else {
+            metric_type = 'Aggregate metric';
+        }
```

Comment:
This can be a ternary and use const
```suggestion
        const metric_type =  $scope.metric.type === 'expression'
            ?  'Expression metric'
            : 'Aggregate metric';
```